src/AppBundle/Controller/MembershipController.php

```
@@ -139,12 +142,43 @@ public function pinInterestsAction(Request $request): Response
     }
 
     /**
+     * This action enables a new user to follow his/her committees.
+     *
      * @Route("/inscription/choisir-des-comites", name="app_membership_choose_nearby_committee")
      * @Method("GET|POST")
      */
-    public function chooseNearByCommitteeAction(): Response
+    public function chooseNearByCommitteeAction(Request $request): Response
     {
-        return new Response('TO BE IMPLEMENTED');
+        $membershipUtils = $this->get('app.membership_utils');
+
+        if (!$id = $membershipUtils->getNewAdherentId()) {
+            throw new AccessDeniedHttpException();
```

Please throw a not found exception like other controllers.